junebug/config.py

```diff
@@ -34,3 +34,7 @@ class JunebugConfig(Config):
             'username': 'guest',
             'password': 'guest'
         })
+
+    ttl = ConfigInt(
+        "Maximum time allowed to reply to messages",
+        default=60)
```

Comment:
I was wondering about that. Should it be the same ttl, or should we have separate ttls?